psa-crypto/src/types/key.rs

```diff
@@ -672,15 +671,14 @@ impl Id {
     }
 }
 
-#[cfg(feature = "with-mbed-crypto")]
```

Comment:
Gotta say, these seem very randomly assigned. For `Attributes` `from_key_id` is activated by the `operations` feature, for `Id` the `from_persistent_key_id` method isn't restricted to any feature.... Not saying there aren't reasons behind this, but it seems very random and I'm really looking forward to the Mbed Crypto version in Mbed TLS to catch up with the spec.......

Also, maybe we should have some mention in the doc what feature each method is gated by - docs.rs will build the default and they don't show what features affect what.

Reply:
> from_key_id is activated by the operations feature, for Id the from_persistent_key_id method isn't restricted to any feature.... 

Because `psa_get_attributes` is an operation that needs linking with the library 😬 

I think all users will use the default, with everything included, Those features are mostly useful for us internally for the SE driver and Parsec interface.